src/replay/execution.zig

```diff
@@ -4,8 +4,6 @@ const replay = @import("lib.zig");
 
 const core = sig.core;
 
-const Allocator = std.mem.Allocator;
```

Comment:
Why did you remove this?

Reply:
Allocator is the one type that I'm always super annoyed about when it doesn't already have an import in a file. Anyway don't worry about it.